q2_feature_table/plugin_setup.py

```diff
@@ -327,6 +327,34 @@
                 "on https://docs.qiime2.org for additional details."
 )
 
+plugin.methods.register_function(
+    function=q2_feature_table.filter_features_conditionally,
+    inputs={'table': FeatureTable[T1]},
+    parameters={'prevelance': Float % Range(0, 1),
+                'abundance': Float % Range(0, 1)},
+    outputs=[('filtered_table', FeatureTable[T1])],
+    input_descriptions={
+        'table': 'The feature table from which features should be filtered.'
+    },
+    parameter_descriptions={
+        'abundance': ('the minimum relative abundance for a feature to be '
+                      'retained'),
+        'prevelance': ('the minimum portion of samples a that a feature '
+                       'must have a relative abundancee of at least '
+                       '`abundance` to be retained')
```

Comment:
```suggestion
        'prevalence': ('the minimum portion of samples that a feature '
                       'must have a relative abundance of at least '
                       '`abundance` to be retained')
```

Reply:
Another minor point here is to begin docstrings with capital letters.

Reply:
Gotta love pep8